src/Symfony/Component/Routing/Matcher/UrlMatcher.php

```diff
@@ -115,6 +120,11 @@ public function addExpressionLanguageProvider(ExpressionFunctionProviderInterfac
         $this->expressionLanguageProviders[] = $provider;
     }
 
+    public function setExpressionConditionVariable(string $name, mixed $value): void
```

Comment:
can we move this to a constructor argument?

Reply:
done via eafc9c5749ac2063e5d9c3689a19202b12cbc6d1